west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 7265d089ed41eeaf0663cab55a3dab2215491d94
+      revision: pull/1468/head
```

Comment:
`f923004850f7ae85f2b02bf339edc69a4e9b0172`

Reply:
Thank you @rlubos . I have added the SHA